packages/components/src/date-time/style.scss

```diff
@@ -46,6 +47,7 @@
 		height: 30px;
 		margin-top: 0;
 		margin-bottom: 0;
+		z-index: 1;
```

Comment:
For `z-index: 1;` cases that simply elevate the item by one, I personally think it's fine to just do this. But I wanted to mention the z-index layer handling mixin we have. Was it intentional to not use it?